panther_power_control/src/power_board_node.py

```diff
@@ -128,45 +124,51 @@ def __init__(self, name: str) -> None:
         #   Timers
         # -------------------------------
 
-        # 2 Hz chearger publishing timer
-        self._charger_state_timer = rospy.Timer(rospy.Duration(0.5), self._publish_charger_state_cb)
-        # 10 Hz e-stop publishing timer
-        self._e_stop_timer = rospy.Timer(rospy.Duration(0.1), self._publish_e_stop_state_cb)
+        # 5 Hz publish non asynch pin state
+        self._charger_state_timer = rospy.Timer(rospy.Duration(0.5), self._publish_pin_state_cb)
```

Comment:
```suggestion
        self._charger_state_timer = rospy.Timer(rospy.Duration(0.5), self._charger_state_cb)
```

Reply:
As discussed since e-stop is also published here we will keep it as it is